Balance/Table Cells/WalletTableViewCell.swift

```diff
@@ -37,13 +29,13 @@ class WalletTableViewCell: UITableViewCell {
         return label
     }()
     
-    private let containerView: UIView = {
+    internal let containerView: UIView = {
```

Comment:
`internal` is not correct here. It essentially does nothing when used outside of a framework as it acts the same as `public` or no modifier at all, `private` is the correct modifier as these are meant to be visible only to this class. Internal only makes sense when you want something to be public, but only inside of a framework.

Reply:
Does `private` allow overriding by subclasses? `renderTableViewCellContentFor`, `nameLabel` and others are set to `internal` for this reason.

Reply:
Ah sorry missed that you were subclassing. In that case just remove the modifier. The default is `internal` so it's superfluous.

Reply:
Changed to `private` since I'm not overriding these properties anyway :)